Source/Mosa.DeviceSystem/FrameBuffer32.cs

```diff
@@ -93,7 +93,7 @@ public void DrawRectangle(uint color, uint x, uint y, uint wi, uint h, uint we)
 	/// <param name="w">Width of the rectangle.</param>
 	/// <param name="h">Width of the rectangle.</param>
 	/// <param name="r">Radius of the corners.</param>
-	public void DrawFilledRoundedRectangle(uint color, uint x, uint y, uint w, uint h, uint r)
+	public void DwarfilledRoundedRectangle(uint color, uint x, uint y, uint w, uint h, uint r)
```

Comment:
It should be ``DrawFilledRoundedRectangle`` here.

Reply:
Fixed.